bundles/org.openhab.binding.intellicenter2/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="intellicenter2"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>IntelliCenter 2 Bridge</label>
+		<description>Pentair IntelliCenter 2 Bridge</description>
+		<properties>
+			<property name="vendor">Pentair</property>
+		</properties>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
+				<context>host</context>
```

Comment:
This is not a valid context, so should be removed. See https://www.openhab.org/docs/developer/bindings/config-xml.html#supported-contexts